src/Nethermind/Nethermind.Network.Test/P2P/PacketSenderTests.cs

```diff
@@ -20,9 +21,10 @@ public class PacketSenderTests
         [Test]
         public void Does_send_on_active_channel()
         {
-            byte[] serialized = new byte[2];
+            IByteBuffer serialized = PooledByteBufferAllocator.Default.Buffer(2);
```

Comment:
NP: Use unpooled by default. Leak is negligable in this case as it only run once.